std_msgs/Header header

sensor_msgs/Image camera_1
sensor_msgs/Image camera_2

nav_msgs/Odometry odom_1
nav_msgs/Odometry odom_2

sensor_msgs/Imu imu_1
sensor_msgs/Imu imu_2